plugins/modules/net_tools/dnsimple_info.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+#  Copyright: Edward Hilgendorf, <edward@hilgendorf.me>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: dnsimple_info
+
+short_description: Pull basic info from DNSimple API
+
+version_added: "1.0.0"
+
+description: This is my longer description explaining my test info module.
+
+options:
+    name:
+        description: The domain name to retrieve info from, will return all associated records
+        required: false
+        type: str
+
+    account_id:
+        description: The account ID to query
+        required: true
+        type: str
+
+    api_key:
+        description: The API key to use
+        required: true
+        type: str
+
+    record:
+        description: The record to find
+        required: false
+        type: str
+
+    sandbox:
+        description: whether or not to use sandbox environment
+        required: false
+        default: false
+        type: bool
+
+author:
+    - Your Name (@yourGitHubHandle)
+'''
+
+EXAMPLES = r'''
+- name: Get all domains from an account
+  dnsimple_info:
+    account_id: "1234"
+    api_key: "1234"
+
+- name: Get all records from a domain
+  dnsimple_info:
+    name: "example.com"
+    account_id: "1234"
+    api_key: "1234"
+
+- name: Get all info from a matching record
+  dnsimple_info:
+    name: "example.com"
+    record: "subdomain"
+    account_id: "1234"
+    api_key: "1234"
+'''
```

Comment:
```suggestion
'''

RETURN = r'''
... add return value documentation here ...
'''
```
This is pretty important for _info modules, so that users know what to expect.